extensions/applicationinsights-dependencies-js/src/TraceParent.ts

```diff
@@ -12,24 +12,29 @@ export class Traceparent {
     private static DEFAULT_TRACE_FLAG = "01";
     private static DEFAULT_VERSION = "00";
     public spanId: string;
-    public traceFlag: string = Traceparent.DEFAULT_TRACE_FLAG;
```

Comment:
Moved this to inside the constructor and referenced as self.xxx as by default TypeScript adds this automatically to it's constructor definition as this.xxxx and the "this." can't be minified.